server/app/routes/api/v1/va/ambassadors.js

```diff
@@ -212,6 +212,25 @@ async function updateW9Ambassdor(req, res) {
   return _204(res)
 }
 
+/*
+ *
+ * updatePayPalApprovedAmbassdaor(req, res)
+ *
+ *
+ */
+async function updatePayPalApprovedAmbassador(req, res) {
+  let found = await req.neode.first('Ambassador', 'id', req.params.ambassadorId)
```

Comment:
Just a note here, we're checking for the existence of this Ambassador. Then below, we're doing the update. We're hitting the database twice.

I think at some point in the future we should just hit the database once with the raw cypher, adding a RETURN of the Ambassador's ID or something. If there are no records, return 404.